packages/addons/addon-depends/network-tools-depends/iperf/patches/iperf-0001_fix-build-with-gcc5.patch

```diff
@@ -10,18 +10,3 @@ diff -Naur a/Makefile.am b/Makefile.am
 +                > $@.tmp
```

Comment:
do we need that patch at all ?
if I remove it it builds fine

Reply:
Removed. You right...